### PR TITLE
As add zumis

### PR DESCRIPTION
**Pull request by sickler-alex**
Created docker for zUMIs 2.9.4.

### CODE REVIEW

File: zumis/2.9.4/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:20.04
+MAINTAINER Alex Sickler (sicklera@email.chop.edu)
+
+ENV ZUMIS_VERSION 2.9.4
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
```
Comment (migbro) [inline zumis/2.9.4/Dockerfile:7]: Perhaps add a comment in the docker file and in the eventual readme to ignore the `./zUMIs.sh: line 7: curl: command not found` warning, as you have explained that it has the undesired effect of trying to upgrade the repo

**Review by migbro — CHANGES_REQUESTED** [review]
Small request, but looks good overall. Perhaps @dmiller15 can comment on the coonfig file format

**Review by migbro — APPROVED** [review]
Looks good, thanks!

**Review by dmiller15 — APPROVED** [review]
Looks good. Although I wonder why those annofuse config statements moved around.

**Pull request merged**